pkg/uiserver/embedded_assets_rewriter.go

```diff
@@ -3,38 +3,41 @@
 package uiserver
```

Comment:
could you also help supplying a license header? I missed this part previously..

Reply:
done